Cargo.toml

```diff
@@ -37,11 +37,13 @@ tokio = { version = "1.25", features = ["rt", "rt-multi-thread"] }
 lazy_static = "1.4.0"
 futures = "0.3.21"
 
-# Rust unoptimized code is so slow that we customize dev profile to enable some
-# optimizations, otherwise Python tests take forever.
+# Rust unoptimized code is really slow with crypto algorithm and regex compilation,
+# hence we optimize our 3rd party dependencies.
+[profile.dev.package."*"]
+opt-level = 1
```

Comment:
why `opt-1`, not `opt-2` ?

Reply:
The difference between `opt-1` and `opt-2` was 20s of increase compilation time for `opt-2` for a gain of `200ms` for executing the test (on @touilleMan computer)